lib/active_merchant/billing/gateways/quickpay/quickpay_v4to7.rb

```diff
@@ -83,7 +83,7 @@ def store(creditcard, options = {})
         post = {}
 
         add_creditcard(post, creditcard, options)
-        add_amount(post, 0, options)
+        add_amount(post, 0, options) if @protocol >= 7
```

Comment:
This change was intentional so that we will send the currency on v3 as well as v7.  See https://github.com/chargify/active_merchant/pull/39